crates/core/src/models/cfg.rs

```diff
@@ -0,0 +1,46 @@
+use std::panic;
```

Comment:
either this or `config.rs` are duplicated, you should probably choose one
additionally, we should try not to use shortened names cfg -> config and StarknetCfg -> StarknetConfig

Reply:
removed it.